test/test_image_scaling.c

```diff
@@ -0,0 +1,39 @@
+#include <stdlib.h>
```

Comment:
You should add speed test, for example: 

**file**: test_image_scaling_benchmark.c 
**output**:

```
method\image size                      1280x800          1920x1080     4096×2160
Graph_Zoom()                           100ms             200ms         380ms
Graph_ZoomBilinear()                   160ms             320ms         500ms
```